notebooks/community/model_garden/model_garden_pytorch_timm.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<p>the Endpoint -&gt; an Endpoint resource</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>